File: ProEvents/Back/src/ProEvents.Domain/Entities/Event.cs
using System;
using System.Collections.Generic;

namespace ProEvents.Domain.Entities
{
    public class Event : Entity
    {
        public string Location { get; set; }
        public DateTime? EventDate { get; set; }
        public string Theme { get; set; }
        public int NumberPeople { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<Batch> Batchs { get; } = new List<Batch>();
        public List<SocialMedia> SocialMedias { get; } = new List<SocialMedia>();
        public List<EventSpeaker> EventSpeakers { get; } = new List<EventSpeaker>();
    }
}

File: ProEvents/Front/ProEvents-App/src/app/models/Event.ts
import { Batch } from './Batch';
import { SocialMedia } from './SocialMedia';
import { Speaker } from './Speaker';

export interface Event {
  id: number;
  location: string;
  eventDate?: Date;
  theme: string;
  numberPeople: number;
  imageURL: string;
  phone: string;
  email: string;
  batches: Batch[];
  socialMedias: SocialMedia[];
  speakers: Speaker[];
  createdAt: Date;
}
